firefox-ios/Client/Frontend/Browser/Tabs/Views/TabTrayViewController.swift

```diff
@@ -265,9 +266,10 @@ class TabTrayViewController: UIViewController,
                 return TabTrayState(appState: appState, uuid: uuid)
             })
         })
-
-        let action = TabTrayAction(windowUUID: windowUUID,
-                                   actionType: TabTrayActionType.tabTrayDidLoad)
+        let action = TabTrayAction(
+            panelType: initialSelectedPanel,
+            windowUUID: windowUUID,
+            actionType: TabTrayActionType.tabTrayDidLoad)
```

Comment:
Just curious is the `panelType` always supposed to be included for action types of `tabTrayDidLoad`?

Reply:
I think it's optional because of this in `tabTrayDidLoad` middleware:

``` 
    private func tabTrayDidLoad(for windowUUID: WindowUUID, panelType: TabTrayPanelType?) {
        let tabManager = tabManager(for: windowUUID)
        let isPrivateModeActive = tabManager.selectedTab?.isPrivate ?? false

        // If no panelType is provided then fallback to whichever tab is currently selected
        let panelType = panelType ?? (isPrivateModeActive ? .privateTabs : .tabs)
        let tabTrayModel = self.getTabTrayModel(for: panelType, window: windowUUID)
        let action = TabTrayAction(tabTrayModel: tabTrayModel,
                                   windowUUID: windowUUID,
                                   actionType: TabTrayActionType.didLoadTabTray)
        store.dispatch(action)
    }
```

But honestly seems like it would be better to make it non-optional in this case. 
Do you have any thoughts @OrlaM ?